util-core/src/main/java/com/indeed/util/core/time/StoppedClock.java

```diff
@@ -7,8 +7,11 @@
 /**
  * Simple wall clock frozen to a moment in time.
  *
+ * @deprecated with {@link WallClock}. Use a mocking framework with a mocked {@link java.time.Clock} instead.
```

Comment:
I'm not sure that we should recommend a mocking framework. It's not really strictly necessary.

Reply:
I think the proper recommendation here is probably java.time.Clock.fixed()

Reply:
Agree that Clock.fixed is a more consistent alternative. I've removed the mocking recommendation, and added that one.